src/queryResult/queryResultWebviewPanelController.ts

```diff
@@ -74,6 +74,18 @@ export class QueryResultWebviewPanelController extends ReactWebviewPanelControll
         this.registerRequestHandler("getWebviewLocation", async () => {
             return qr.QueryResultWebviewLocation.Document;
         });
+        this.registerRequestHandler("getFontFamily", async () => {
+            console.log("getFontFamily");
+            return this._vscodeWrapper
+                .getConfiguration(Constants.extensionName)
+                .get(Constants.extConfigResultKeys[3]);
+        });
+        this.registerRequestHandler("getFontSize", async () => {
+            console.log("getFontFamily");
```

Comment:
looks like we're logging the wrong function name?  Maybe we should use the logger class rather than console calls?

Reply:
Yeah, this logging was just for my debugging, I'll be removing it before I merge the PR and I will add more detailed logs using the logger class in the future